ERROR - 2023-10-19 14:54:45 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'mall' in C:\laragon\www\Lab3\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\Lab3\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'mall', 3306, '', 0)
#1 C:\laragon\www\Lab3\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\Lab3\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\Lab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\Lab3\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\Lab3\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\Lab3\app\Controllers\ProductController.php(101): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\Lab3\system\CodeIgniter.php(919): App\Controllers\ProductController->home()
#8 C:\laragon\www\Lab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\laragon\www\Lab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\Lab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\Lab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'mall' in C:\laragon\www\Lab3\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\Lab3\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\Lab3\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\Lab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\Lab3\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\Lab3\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\Lab3\app\Controllers\ProductController.php(101): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\Lab3\system\CodeIgniter.php(919): App\Controllers\ProductController->home()
#7 C:\laragon\www\Lab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 C:\laragon\www\Lab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\Lab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\Lab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2023-10-19 14:54:45 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'mall'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `lamall`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProductController.php(101): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\Lab3\\public\\index.php')
ERROR - 2023-10-19 15:09:47 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'mall' in C:\laragon\www\Lab3\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\Lab3\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'mall', 3306, '', 0)
#1 C:\laragon\www\Lab3\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\Lab3\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\Lab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\Lab3\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\Lab3\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\Lab3\app\Controllers\ProductController.php(101): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\Lab3\system\CodeIgniter.php(919): App\Controllers\ProductController->home()
#8 C:\laragon\www\Lab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\laragon\www\Lab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\Lab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\Lab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'mall' in C:\laragon\www\Lab3\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\Lab3\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\Lab3\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\Lab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\Lab3\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\Lab3\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\Lab3\app\Controllers\ProductController.php(101): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\Lab3\system\CodeIgniter.php(919): App\Controllers\ProductController->home()
#7 C:\laragon\www\Lab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 C:\laragon\www\Lab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\Lab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\Lab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2023-10-19 15:09:47 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'mall'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `lamall`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProductController.php(101): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\Lab3\\public\\index.php')
